server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/main/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultRoutes.java

```diff
@@ -109,7 +111,10 @@ public String getValue() {
     public static final String USERS = "users";
     public static final String USER_PATH = ROOT_PATH + SEPARATOR + USERS;
     private static final String USER_PATH_PARAM = "user";
```

Comment:
Why not using the same pattern as `MESSAGE_ID_PARAM` variable, as : 
`USER_PATH_PARAM = ":user"` instead?